src/demo/table-demo.tsx

```diff
@@ -111,6 +112,16 @@ export function TableDemo() {
         isError: false,
         handleExport: () => {},
       }}
+      dateRangeConfig={{
+        onChange: (value: DateRange) => {
+          console.log({testing: value})
+        },
+        value: {
```

Comment:
keep the initial values inside the tableStore, we need to treat it similar to other filters